descriptors/ModuleDescriptor-template.json

```diff
@@ -1116,6 +1116,27 @@
           "permissionsRequired": ["inventory-storage.inventory-hierarchy.items-and-holdings.collection.post"]
         }
       ]
+    },
+    {
+      "id": "instance-reindex",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/instance-storage/reindex",
+          "permissionsRequired": ["inventory-storage.instance.reindex.post"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/instance-storage/reindex/{id}",
+          "permissionsRequired": ["inventory-storage.instance.reindex.item.get"]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/instance-storage/reindex/{id}",
+          "permissionsRequired": ["inventory-storage.instance.reindex.item.delete"]
+        }
```

Comment:
Are these endpoints for management of a specific reindexing job?

Reply:
yes

Reply:
What happens if more than one reindexing job runs at the same time? 

As I understand it, the re-index job will publish messages to the general `inventory.instances` topic, do all of the clients know how to ignore these if they aren't interested or discriminate messages for a particular job if they process them?